plugins/package-managers/python/src/main/kotlin/Pipenv.kt

```diff
@@ -86,7 +86,10 @@ class Pipenv(
 
         requirementsFile.writeText(requirements)
 
-        return Pip(managerName, analysisRoot, analyzerConfig, repoConfig)
+        val pipenvAnalyzerConfig = analyzerConfig
+            .withPackageManagerOption(managerName, "overrideProjectType", projectType)
+
+        return Pip(managerName, analysisRoot, pipenvAnalyzerConfig, repoConfig)
```

Comment:
out of scope: Is it actually fine to create a new instance of Pip each time? (IIRC dependency graph building relies on using only once instance)

Reply:
Hm, I believe I was after a different requirement, was it that same `DependencyGraphBuilder` or `Handler` instance needs to be used, maybe?

Reply:
No idea...